src/config/autoload/develop.local.php.dist

```diff
@@ -29,5 +29,9 @@ return [
                'ttl' => 1
             ]
         ]
-    ]
+    ],
+    // 'assets_host'     => 'http://localhost:8081/',
+    // 'editor_renderer' => [
+    //   'url' => 'http://host.docker.internal:3000/',
```

Comment:
@Knorrke This works in Docker for Mac. Maybe for Windows, too (or something similar)

Reply:
nice, thanks! Looks promising, the docs of docker for windows say this should work. Will test this tomorrow